dangerzone/gui/main_window.py

```diff
@@ -296,6 +296,12 @@ def __init__(self, dangerzone: DangerzoneGui) -> None:
         super(SettingsWidget, self).__init__()
         self.dangerzone = dangerzone
 
+        # Num Docs Selected
+        self.docs_selected_label = QtWidgets.QLabel("No documents selected")
+        self.docs_selected_label.setAlignment(QtCore.Qt.AlignCenter)
+        self.docs_selected_label.setContentsMargins(0, 0, 0, 20)
+        self.docs_selected_label.setProperty(b"class", "docs-selection")
```

Comment:
I see something weird here. If I run it as is, I get the following errors:

```
FIXME typing.Any cannot be used with isinstance()
...
TypeError: 'PySide2.QtCore.QObject.setProperty' called with wrong argument types:
  PySide2.QtCore.QObject.setProperty(bytes, str)
Supported signatures:
  PySide2.QtCore.QObject.setProperty(bytes, typing.Any)
QThread: Destroyed while thread is still running
Aborted (core dumped)
```

If I replace `b"class"` with `"class"`, then I don't get any error. Any particular reason you opted to use bytes there? I'm using Python 3.10.6, if it helps.

Reply:
Oh. That was a typo. I must have changed to `b"class"` last minute to see if it would fix the typing, but I forgot about it. 

It will have to be string and with `# type: ignore` comment. We've done it somwhere else in the code. I couldn't find a way to satisfy the linter.

Reply:
I added a FIXUP commit to address this.